admin/CR-vcdm/Approval_request.md

```diff
@@ -1,64 +1,90 @@
-(Draft for a CR Transition DI, cryptosuites, json schemas)
-
+(Draft for a CR transition of VCDM v2.0 and VC Specifications Directory)
 
 # Document title, URLs, estimated publication date
 
 - Verifiable Credentials Data Model v2.0
-  - https://www.w3.org/TR/2023/CR-vc-data-model-2.0-202312@@/
-  - @@ December 2023
+  - https://w3c.github.io/vc-data-model/CR/2023-12-21/
+  - 21 December 2023
   - Editors' draft: https://w3c.github.io/vc-data-model/
 
 # Abstract
 
-- https://www.w3.org/TR/2023/CR-vc-data-model-2.0-202312@@/#abstract
+- https://w3c.github.io/vc-data-model/CR/2023-12-21/#abstract
 
 # Status
 
-- https://www.w3.org/TR/2023/CR-vc-data-model-2.0-202312@@/#sotd
-  
+- https://w3c.github.io/vc-data-model/CR/2023-12-21/#sotd
+
 # Link to group's decision to request transition
 
 - https://www.w3.org/2017/vc/WG/Meetings/Minutes/2023-12-@^@#resolution@^^@
 
 # Changes
 
-- https://www.w3.org/TR/2023/CR-vc-data-model-2.0-202312@@/#revision-history
-- 
+- https://w3c.github.io/vc-data-model/CR/2023-12-21/#revision-history
+
 # Requirements satisfied
 
 Yes.
 
 # Dependencies met (or not)
 
+All normative dependencies are either RECs or IETF RFCs, except for:
+
+* Verifiable Credential Data Integrity, which entered CR in November 2023.
+* Securing Verifiable Credentials using JOSE and COSE, which is a WD and expected to enter CR in Q1 2024.
```

Comment:
In view of https://github.com/w3c/vc-data-model/pull/1338 [VC-SPECS] may also join this list as a 'draft note' (unless we decide to publish it as a final note right away).

Reply:
The goal is to publish that document as a Note. By the time the VCDM is requested to CR this should be done. This comment is therefore moot.